### DISCUSSION TITLE
why this logged?

### DISCUSSION THREAD

**Discussion by aafnnp** [vercel/next.js#42561] [Help]
![image](https://user-images.githubusercontent.com/12791216/200239827-a2981bb9-6f06-45c4-9fb4-a8a9bc4a39ea.png)

**Comment by icyJoseph** [comment]
It might be that the SWC you have in the lock file has been compiled for a different system than the one you use. SWC is written in rust, so it ought to be compiled for your target architecture.

Did you run `npm install` or `npm ci`?

**Reply by aafnnp** [reply]
appeared while reinstall without lock file

**Reply by icyJoseph** [reply]
That would make sense, since these are optional dependencies, and there's no lock file, it is trying to figure out if they apply to your architecture. No lock file is present to restrict them. This used to happen a lot with `fsevents` which was needed for MacOS users. If you had a Linux/Windows Machine and installed something that needed file system you'd see `fsevents` not being installed warnings.
